.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: klaytn-safe-react deploy workflow
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  deploy:
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_ARN_DEV }}
+          role-session-name: SessionForKlaytnActions
+          aws-region: ${{ secrets.AWS_REGION_DEV }}
+      - name: Installing Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "14"
+      - name: Installing dependencies
+        run: yarn install
+      - name: Change homepage and get .env
+        env:
+          PARAMETER_STORE_DEV: ${{ secrets.PARAMETER_STORE_DEV }}

```

Comment:
For what do we need parameter store?

Reply:
What kinds of static files are stored in parameter store? Please show me the data in `.env`.

Reply:
@Sotatek-TinnNguyen  All values stored in `.env` are not required. `REACT_APP_GATEWAY_URL` and `REACT_APP_ENV` can be stored by GitHub Action Secrets as we discussed.